### ISSUE TITLE
Allow definition on "security" or enable inclusion of parent openapi definition

### ISSUE THREAD

**Issue by cljk** [openapi-tools/swagger-maven-plugin#46]
**Is your feature request related to a problem? Please describe.**
My API definition should contain reference to the used API key like
```
security:
- ApiKeyAuth: []

...

  securitySchemes:
    ApiKeyAuth:
      type: apiKey
      name: X-API-KEY
      in: header
```
This seems not to be possible with the current plugin.

**Describe the solution you'd like**
Possible solutions - I like both ;-)
* enable usage of an appendable openapi definition which could contain this definition
* enable definition of `securitySchemes` and `security` in `pom.xml`

**Comment by wanno-drijfhout** [comment]
Should be solved with in #53

**Comment by langecode** [comment]
#53 (replaced by #58) will be included in next release (this week).
